org-formation/050-costs/program-code-rules.json

```diff
@@ -0,0 +1,399 @@
+[
+  {
+    "Type": "REGULAR",
+    "Value": "Platform Infrastructure",
+    "Rule": {
+      "And": [
+    {
+      "Dimensions": {
+        "Key": "LINKED_ACCOUNT",
+        "Values": [
+          "531805629419",
+          "140124849929",
+          "254940560790",
+          "231505186444",
+          "153370007719",
+          "087547153226",
+          "797640923903",
+          "867686887310",
+          "420786776710",
+          "745159704268",
+          "325565585839",
+          "383874245509",
+          "449435941126",
+          "649232250620",
+          "804034162148",
+          "743644221192"
+        ],
+        "MatchOptions": [
+          "EQUALS"
+        ]
+      }
+    },
+    {
+      "Tags": {
+        "Key": "CostCenter",
+        "MatchOptions": [
+          "ABSENT"
+        ]
+      }
+    },
+    {
+      "Tags": {
+        "Key": "CostCenterOther",
+        "MatchOptions": [
+          "ABSENT"
+        ]
+      }
+    }
+      ]
+    }
+  },
+  {
+    "Type": "REGULAR",
+    "Value": "Platform Infrastructure",
+    "Rule": {
+      "Tags": {
+    "Key": "CostCenter",
+    "Values": [
+      "Platform Infrastructure"
+    ],
+    "MatchOptions": [
+      "STARTS_WITH"
+    ]
+      }
+    }
+  },
+  {
+    "Type": "REGULAR",
```

Comment:
I haven't been able to find a way, I suspect because the tags are on the organization, not the account, and the two are slightly different; but I've opened a feature request for AWS to propagate the organizations tags to resources within the organization in cost explorer.